client/components/Item/Card/Card.vue

```diff
@@ -214,7 +216,9 @@ export default Vue.extend({
       return this.getItemDetailsLink(this.item);
     },
     /*
```

Comment:
This needs to be `/**` otherwise it's not valid JSDOC.

Reply:
I was wondering why the coloration didn't work